### PR TITLE
added homepage and css

### PR DESCRIPTION
**Pull request by Lacko3006**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -1,43 +1,104 @@
...
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (kennywenny) [inline client/public/index.html]: Probably wise not to change the index.html: add the starter HTML to the App component instead.

File: client/src/App.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import Homepage from "./components/Homepage"
 function App() {
   return (
    <div>
-      <Homepage />
+    
+<Homepage />
```
Comment (kennywenny) [inline client/src/App.js]: Your spacing is off on this.

File: package.json [hunk 3]
```diff
@@ -23,5 +23,8 @@
   "dependencies": {
     "concurrently": "^7.6.0",
     "dotenv": "^16.0.3"
+  },
+  "devDependencies": {
```
Comment (kennywenny) [inline package.json]: Why do you need webpack?

**Review by kennywenny — CHANGES_REQUESTED** [review]

File: client/public/index.html (hunk 1 above)
Comment (Lacko3006) [reply client/public/index.html]: this has been altered.

File: client/src/App.js (hunk 2 above)
Comment (Lacko3006) [reply client/src/App.js]: this has since been changed.

File: package.json (hunk 3 above)
Comment (Lacko3006) [reply package.json]: I have now deleted this due to not being needed.

**Comment by Lacko3006** [comment]
All changes made and new ones prompted.

**Review by kennywenny — APPROVED** [review]

**Pull request merged**